.github/scripts/boost/build_and_test_boost.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+BOOST_BUILD_DIR=$1
+BOOST_TEMPLATE_DIR=$2
+
+if [[ $# -lt 1 ]]; then
+    echo "Expected at least 1 arguments!" >&2
+    exit 1
+fi
+
+if [[ -d $BOOST_BUILD_DIR ]]; then
+    echo "Directory $BOOST_BUILD_DIR already exists!" >&2
+    exit 1
+fi
+
+mkdir -p $BOOST_BUILD_DIR
+
+if [[ ! -z $BOOST_TEMPLATE_DIR && -d $BOOST_TEMPLATE_DIR ]]; then
+    echo "Found Boost template dir '$BOOST_TEMPLATE_DIR', using it instead of cloning.."
+    time cp -r $BOOST_TEMPLATE_DIR/* $BOOST_BUILD_DIR
+else
+    echo "Cloning Boost GitHub repository.."
+    time git clone --recursive https://github.com/boostorg/boost.git $BOOST_BUILD_DIR
+    # process module contained bug which prevented building it with MinGW, there's already fix
+    # but it's not updated in Boost project yet.
+    cd $BOOST_BUILD_DIR/libs/process
+    git checkout develop
+    cd -
+fi
+
+cd $BOOST_BUILD_DIR
+
+echo "using gcc :  : /home/vejby/cross-aarch64-w64-mingw32-msvcrt/bin/aarch64-w64-mingw32-g++ ;" > "user-config.jam"
+
+echo "Running Boost bootstrap.."
```

Comment:
Please use https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#grouping-log-lines even when this script cannot be used in GitHub Actions for now for consistency with the rest of the scripts